vumi/tests/helpers.py

```diff
@@ -1407,14 +1407,25 @@ def get_riak_manager(self, config=None):
             Riak manager config. (Not a complete worker config.) If ``None``,
             the one used by :meth:`mk_config` will be used.
 
+        :param bool prepend_bucket_prefix:
+            If ``True`` (the default), the ``bucket_prefix`` field in the given
+            config (if any) will have the test prefix prepended to it. To keep
+            the ``bucket_prefix`` unmodified, set this to ``False``.
+
         :returns:
             A :class:`~vumi.persist.riak_manager.RiakManager` or
             :class:`~vumi.persist.riak_manager.TxRiakManager`, depending on the
             value of :attr:`is_sync`.
         """
         self._check_patches_applied()
+        default_config = self._config_overrides["riak_manager"].copy()
         if config is None:
-            config = self._config_overrides['riak_manager'].copy()
+            config = default_config
+        else:
+            config = config.copy()
+            if prepend_bucket_prefix:
```

Comment:
Could we just leave the option out for now and add it back if we really
need it? I think any tests currently relying on the old behaviour should be
fixed since having a bucket prefix outside of the test one seems like it
might cause problems if, e.g., riak on a dev's machine has other data in it.